src/v3/src/transformer/profile/transformEnrollProfile.ts

```diff
@@ -51,7 +51,6 @@ export const transformEnrollProfile: IdxStepTransformer = ({ transaction, formBa
         userInfo: getUserInfo(transaction),
         settings: passwordSettings,
         requirements: buildPasswordRequirementListItems(passwordSettings),
-        fieldKey: 'credentials.passcode',
```

Comment:
why it's here before?

Reply:
Previously (Pre Generic Remediator) the password field had more than 2 possible values, now that is is only either `credentials.passcode` or `credentials.newPassword` there is no need to pass this value to the requirements component, the requirements component can conditionally try either of those fields to run validation on. So I removed it since it is no longer needed.